### ISSUE TITLE
Creating document result in an error

### ISSUE THREAD

**Issue by dimityrivanov** [appwrite/sdk-for-flutter#3]
Using this piece of code to add document it result in an error also into the /console I am unable to find button to add document to collection.
What am I doing wrong ?
```
    Future result = database.createDocument(
      collectionId: '<my id was here>',
      data: [ // Rules
        {
          "label": "Name",
          "key": "name",
          "type": "text",
          "default": "Empty Name",
          "required": true,
          "array": false
        },
        {
          "label": "Release Year",
          "key": "releaseYear",
          "type": "numeric",
          "default": 1970,
          "required": true,
          "array": false
        },
        {
          "label": "Notes",
          "key": "notes",
          "type": "text",
          "default": null,
          "required": false,
          "array": true
        }
      ],
      read: [],
      write: [],
    );
```

**Comment by eldadfux** [comment]
How does your collection rules look like?
What exact error are you getting?

Your payload looks like a payload for creating a new collection, but you are trying to create a new document, is this intentional? 

The examples in this page can also help you: https://appwrite.io/docs/rules

**Comment by dimityrivanov** [comment]
I just created a simple collection it was created without rules so I just left it that way, when I select the collection in the console I wasn't able to find a button to add a document so I decided to try from the plugin the output from the above code is: `flutter: DioError [DioErrorType.RESPONSE]: Http status error [404]`.

**Comment by eldadfux** [comment]
There's an open bug we already fixed for upcoming version (0.6.2) where document creation is only available to registered user 😒

**Comment by dimityrivanov** [comment]
😒
